src/protocol/cEMI/LDataInd.ts

```diff
@@ -67,7 +73,15 @@ export default class LDataInd extends CEMIMessage {
 			controlField.addressType,
 		)
 		offset += dstAddress.length
-		const npdu = NPDU.createFromBuffer(buffer, offset)
+		let npdu
+		try {
+			npdu = NPDU.createFromBuffer(buffer, offset)
+		} catch (error) {
+			sysLogger.error(
+				`createFromBuffer: ${error.message} srcAddress: ${srcAddress}, dstAddress: ${dstAddress}`,
+			)
+		}
```

Comment:
I think it may be better to catch this error on knxclient side or is the frame still valid if the npdu decoding fails?

Reply:
Hi!
the error catched by the NPDU lacks of source and destination addresses, so i had to implement the error trapping in the LDataInd. Fatto di corsa. The error is due to a KNX Secure telegram sent by an MDT SCN-Safe.01 which cyclically queries other physical KNX addresses (actuators/sensors) to see whether they can still be reached (heartbeat).

Reply:
Ok :)